var/spack/repos/builtin/packages/qucs/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+import os
+
+
+class Qucs(AutotoolsPackage):
+    """QUCS - Quite Universal Circuit Simulator:
+       Qucs is an integrated circuit simulator which means you are able to
+       setup a circuit with a graphical user interface (GUI) and simulate
+       the large-signal, small-signal and noise behaviour of the circuit.
+       After that simulation has finished you can view the simulation results
+       on a presentation page or window.
+       This is the complete environment as provided at Sourceforge, the main repository.
+    """
```

Comment:
The first paragraph is used as a short description, while the entire docstring acts a long description in many contexts:
```suggestion
    """QUCS - Quite Universal Circuit Simulator

    Qucs is an integrated circuit simulator which means you are able to
    setup a circuit with a graphical user interface (GUI) and simulate
    the large-signal, small-signal and noise behaviour of the circuit.
    After that simulation has finished you can view the simulation results
    on a presentation page or window.
    """
```